bin/dmarc_send_reports

```diff
@@ -101,9 +101,9 @@ sub send_single_report {
 
     alarm($alarm_at);
 
```

Comment:
That would be better yes, would make it easier to add more send methods in the future too.